src/qt/window.cpp

```diff
@@ -251,11 +248,14 @@ wxWindowQt::~wxWindowQt()
 
     DestroyChildren(); // This also destroys scrollbars
 
-    delete m_qtPainter;
-
 #if wxUSE_ACCEL
-    m_qtShortcutHandler->deleteLater();
-    delete m_qtShortcuts;
+    if ( m_qtShortcuts )
+    {
+        for ( int i = 0; i < m_qtShortcuts->size(); ++i )
+        {
+            delete m_qtShortcuts->at(i);
```

Comment:
Very minor, of course, but is there any reason not to use just `[]` here? I think in a simple loop from `0` to `size()`, it's acceptable to just use non-checked `operator[]` instead of `at()`.

Reply:
Just that it looked slightly uglier to write `m_qtShortcuts->operator[](i)`.
But then again, maybe the smart pointer could also be replaced with a `wxVector` instance?

Reply:
Why not `(*m_qtShortcuts)[i]` which is not too bad IMO?